README.md

```diff
@@ -252,6 +254,7 @@ Asynchronous concurrent distributed networking framework for PHP
 
 ### Promises
 
+* [amphp/amp](https://github.com/amphp/amp) - A non-blocking concurrency framework for PHP applications
```

Comment:
I agree, but as I said, the promises implementation isn't independent. I'm trying to avoid listing items in multiple sections primarily because I think it could result in a lot of redundancy and make this list a lot more difficult to manage.

I've been wondering if a simple categorized listing is necessarily the best format for this repository. I'm trying to balance ease of editing / contribution with ease of deploying updates while still maintaining optimal structuring / organization of the data. I'm open to suggestions if you think there's something better suited to the purpose.